Atomic/diff.py

```diff
@@ -43,8 +43,12 @@ def diff(self):
             # Clean up
```

Comment:
Somewhere in diff.py, you will need to make an association between self.is_dbus and setting self.args.json (or make a new var).  In other words, you always want to output JSON if is_dbus.